examples/references.bib

```diff
@@ -387,6 +387,12 @@ @article{spiller2013spotlights
   year          = {2013},
   publisher     = {SAGE Publications Sage CA: Los Angeles, CA}
 }
+@article{stucchio2015bayesian,
```

Comment:
I think article type references in bibtex need a journal. The reference is not rendering correctly even though the key looks right and exists,  and I think this is the reason. Preview: https://pymcio--383.org.readthedocs.build/projects/examples/en/383/case_studies/bayesian_ab_testing_introduction.html#references

Reply:
Thanks! I've changed the reference type to `online` since this is a whitepaper that didn't get published in a journal. Similar to the `vandenbergSPSS` reference.